test/index.spec.js

```diff
@@ -184,6 +202,110 @@ describe('AMQP mixin', () => {
         return errorStrategy.should.have.been.calledOnce;
       });
     });
+
+    describe('rpc', () => {
+      describe('publish to exchange', () => {
+        before('bind queue to exchange',
+          () => service.channel.bindQueue('simple', 'direct', 'routingKey'));
+
+        before('rpc', () => {
+          service.rpc(
```

Comment:
maybe it is possible to set a handler on `publish` or `sendToQueue` and invoke channel callback with `yields`? and you don't need to wait for 500

Reply:
Removed "wait"s, now it's better, thanks for the advice